src/windows/process.rs

```diff
@@ -712,42 +720,119 @@ fn check_sub(a: u64, b: u64) -> u64 {
         a - b
     }
 }
-
-pub(crate) fn compute_cpu_usage(p: &mut Process, nb_processors: u64, now: ULARGE_INTEGER) {
+///Before changing this function, you must consider the following:
+/// https://github.com/GuillaumeGomez/sysinfo/issues/459
+pub(crate) fn compute_cpu_usage(p: &mut Process, nb_processors: u64, _now: ULARGE_INTEGER) {
     unsafe {
-        let mut sys: ULARGE_INTEGER = std::mem::zeroed();
-        let mut user: ULARGE_INTEGER = std::mem::zeroed();
+        //the 500 ms can be changed to another value but not in a completly arbitrary way, you must check that the new value does not produce this issue:
+        // https://github.com/GuillaumeGomez/sysinfo/issues/459
+        if p.cpu_calc_values.last_call.elapsed().as_millis() < 500{
```

Comment:
That's exactly what I didn't want. :)

Reply:
I know, me neither :( but i couldn't find other way of making it work. even with that delay when the cpu usage is close to the maximum you might get a value up to 101.5% over 100%. Adding a delay of 1 second solves that but i think that this is far better, even if it has one outliner sometimes.

Reply:
Maybe check [this](https://github.com/heim-rs/heim/commit/060a131e7311b5421695f326f188907b9cb2cdb1) from heim?
Unfortunatly i might not have the time to study it soon (maybe in the weekend)

Reply:
Yes but it's up to the user to ensure that they're not calling the refresh like crazy. I think it'd be worth to mention that in the method description instead of forcing it like that. I'll take your commits and add a fix on top of them because they improve the situation already. We can try to improve the situation even further afterwards.